Update requirements, add web apps

manics opened the pull request:
- Updates requirements
- Adds some web apps with configuration
- Removes use of `hosts: all`

This is intended as a reference for the OME2020 meeting.

Alternatively this could go into another repository

manics committed:
Update requirements.yml

manics committed:
Update playbook, add figure iviewer parade mapr

manics committed:
Add comments about ubuntu 18.04

manics committed:
Update to remove "hosts: all"

Add example inventory

manics committed:
Disable yamllint comments-indentation

manics commented on requirements.yml:
```diff
@@ -1,13 +1,13 @@
 
 - src: ome.omero_server
-  version: 3.0.0
+  version: 3.2.0
```
```suggestion
  version: 3.3.0
```
As soon as Ansible galaxy updates...

manics committed:
Update requirements.yml

sbesson commented:
My  2p about the addition of the various apps  https://github.com/ome/omero-deployment-examples#public-user initially restricted the scope of this example to recreate the configuration of the public OMERO user configuration.

From my side,  there is definitely value in maintaining a minimal example playbook deploying a more advanced OMERO ecosystem (web apps, plugins, maybe micro-services) though. If this were to be constructed as a new repository, what would be your thoughts in terms of scope and name?

manics commented:
If you see microservices being added too how about `ansible-example-omero+addons`?

joshmoore commented:
`s/+/-/` but otherwise that would work for me. Would certainly give us somewhere to explore the micro services.

jburel commented:
``omero_addons`` sounds good

joshmoore commented:
> omero_addons

Now it's almost sounding like a role...

manics commented on requirements.yml:
```diff
@@ -1,13 +1,13 @@
+  version: 3.1.0
 
 - src: ome.omero_user
   version: 0.1.2
```
```suggestion
  version: 0.2.0
```

manics committed:
Update requirements.yml

manics committed:
Add ubuntu test scenario

manics committed:
Ensure openssl is installed

manics committed:
Fix tests

manics committed:
Changes for a new repo

- Rename omero-public omero-addons
- Change license date since this will be a new repo
- Add README

manics commented:
Created a new repo https://github.com/manics/ansible-example-omero-addons
